workshops/personal_website/README.md

```diff
@@ -0,0 +1,575 @@
+  - Is your **username** entered correctly
+  - The repository name should end in **`.io`**
+5. After you are sure you entered in the repository name correctly, click the
+   green "Create repository" button at the bottom.
+6. Then copy the **"HTTPS link"** at the top. It should look like this:
+   `https://github.com/username/username.github.io.git`. You will need this link
+   for the next step.
+
+### 3) Sign Up for Cloud9
+
+Just like you can use Google Drive to write and organize documents written in
+English, we'll be using **Cloud9** to write, save, and organize our code.
+
+> The techy term for a place where you can write, and organize code is **"IDE"**
+> or "Integrated Development Environment"**
+
+1. In a new tab, open https://c9.io/
+2. In the top right hand corner, click the button that looks like this:
+   ![](img/c9_gh_icon.png)
+3. Click the green **"Authorize application"** button
+4. If there is a popup that asks for your email, go ahead and enter it
+5. Click the gray box that says **"Create a new workspace"**
+6. Set the **"Workspace name"**, to `projects`
+   is your GitHub username.
+7. Under **"Clone from Git or Mercurial URL"**, paste the **HTTPS Link** that
+   you copied your GitHub repository.
+
+   > ![](img/c9_add_git_url.gif)
+
+8. Then without changing anything else on the screen, scroll to the bottom and
+   click the green **"Create Workspace"** button.
+
+You should now see a screen looks something like this:
+
+![](img/c9_signup_ide_loaded.png)
+
+> Note:
+>
+> If you're waiting for a while (more than 10 seconds) and the above screen
+> still doesn't load, try the following:
+>
+> 1. Open https://c9.io
+> 2. Click on the green **"Open"** button**
+> 3. You should now see screen shown above. If not, ask your facilitator for
+>    help
+
+### 3) Celebrate!
+
+> ![](img/celebrate_king_kid.gif)
+
+Congratulations, you've officially set up all of your coding tools for the
+semester!
+
+Now we're ready to start the website!
+
+## Part II: Create the Project Folder
+
+We're going to create a new folder for each project in Hack Club.
+
+Let's create a folder for our personal website project:
+
+1. On the left side of the screen, right click the **`projects`** folder
+2. Click **"New Folder"**
+3. Then name the folder **`personal_website`**
+
+> ![](img/c9_create_personal_website_folder.gif)
+
+## Part III: HTML
+
+Every website is written with **HTML** code. HTML code is written in **HTML
+files**.
+
+> HTML stands for Hypertext Markup Language.
+
+### 1) Create the HTML file for your website
+
+1. Right click the **`personal_website`** folder you just created
+2. Click **"New File"**
+3. Then name the file **`index.html`**
+4. Double click it to open it
+
+> ![](img/c9_create_index_dot_html.gif)
+
+> Notes:
+>
+> - Just like Microsoft Word Documents end in **`.doc`**, HTML files end in
+**`.html**`
+> - Your main HTML file has to be named **`index.html`**
+
+### 2) Adding standard HTML template
```

Comment:
In general the workshop is pretty inconsistent in terms of header capitalization. I personally don't think it's a big deal. Is this one of the things that investors kept pointing out (in addition to spelling errors)?